### PR TITLE
IAL2 billing

### PR DESCRIPTION
**Pull request by stevegsa**
**How**: Add an entry for total unique IAL2s for the fiscal year (total proofed users).  Add a new parameter to the report invocation which is the feed for fiscal active users report  https://github.com/18F/identity-idp/pull/3612

### CODE REVIEW

**Comment by stevegsa** [comment]
## Screenshot ##

[billing-sample.pdf](https://github.com/18F/identity-billing-reporter/files/4313296/billing-sample.pdf)

**Comment by stevegsa** [comment]
## Screenshot ##

Screen Shot 2020-03-10 at 11 37 47 AM

**Review by zachmargolis — APPROVED** [review]
LGTM, do we have any tests for this?

**Comment by stevegsa** [comment]
@zachmargolis we do now :-)

File: spec/integration/billing_reporter_spec.rb [hunk 1]
```diff
@@ -8,8 +8,9 @@
           dest_dir: '.',
           year: 2020,
           month: 1,
-          auths_json: 'spec/fixtures/total_monthly_auths.json',
           sp_yml: 'spec/fixtures/service_providers.yml',
+          auths_json: 'spec/fixtures/total_monthly_auths.json',
+          active_users_json: 'spec/fixtures/total_monthly_auths.json',
```
Comment (zachmargolis) [inline spec/integration/billing_reporter_spec.rb:13]: ❤️ for a future PR can we also add CircleCI configs?

Comment (stevegsa) [reply spec/integration/billing_reporter_spec.rb:13]: Sure.

**Pull request merged**